VerifyEncoding/Test-Encoding.ps1

```diff
@@ -89,14 +100,33 @@ function Test-Encoding
                 continue
             }
 
+            $fileExists = Test-Path -Path $file
+
+            if ($fileExists -eq $False)
+            {
+                Write-Host "File $file is deleted. Skipping ..."
+                continue
+            }
+
+            $fileIsFolder = (Get-Item $file).PSIsContainer
```

Comment:
Believe it or not, but the right way is `Get-Item -Force -LiteralPath $file`.

`-LiteralPath` for the reason described above.

`-Force` to include hidden files (such as files ending with dot on Unix).